src/Dhadgar.Scope/src/components/layout/SectionNav.tsx

```
@@ -1,79 +1,79 @@
+        ) : (
+          <div />
+        )}
+
+        {nextSection ? (
+          <a
+            href={`/s/${nextSection.slug}`}
+            className="flex items-center gap-2 rounded-xl border border-white/10 bg-white/5 px-3 py-2 text-sm transition-colors hover:bg-white/10"
+          >
+            <span className="hidden sm:inline">{nextSection.title}</span>
+            <span className="sm:hidden">Next</span>
+            <svg className="h-4 w-4" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)